powersimdata/network/model.py

```diff
@@ -39,3 +38,78 @@ def _import_constants(self, kind):
         """
         mod = import_module(f"powersimdata.network.{self.model}.constants.{kind}")
         return {a: getattr(mod, a) for a in dir(mod)}
+
+    def area_to_loadzone(self, *args, **kwargs):
+        """Map the query area to a list of loadzones, using the known grid model."""
+        return area_to_loadzone(
+            self.model, *args, mappings=self.zones.mappings, **kwargs
```

Comment:
Should not it be `mappings=self.zones["mappings"]`?

Reply:
You are right. Fixed, and now it works properly:
```python
>>> from powersimdata.network.model import ModelImmutables
>>> mi = ModelImmutables("usa_tamu")
>>> mi.area_to_loadzone("Colorado")
{'Colorado'}
```